tests/__mocks__/fileMock.js

```diff
@@ -0,0 +1 @@
+module.exports = 'test-file-stub';
```

Comment:
would be nice to get rid of that if all possible

Reply:
We could remove it, but then the user could not test files that import anything but js/ts :/

Reply:
Actually, after doing some googling, others have also used the identity-obj-proxy for both css and files